packages/mdc-layout-grid/_variables.scss

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+$mdc-layout-grid-breakpoints: (
+  desktop: 840px,
+  tablet: 480px,
+  phone: 0px
+);
+
+$mdc-layout-grid-columns: (
+  desktop: 12,
+  tablet: 8,
+  phone: 4
+);
+
+$mdc-layout-grid-default-column-span: 4;
```

Comment:
I'll add the `!default`, but I prefer not to document this, as it seems like a very error-prone / hard-to-debug-later way of doing things.